test/libsolidity/syntaxTests/constants/define_constants_with_external_constant_member_var.sol

```diff
@@ -0,0 +1,11 @@
+uint256 constant MAX = 1;
```

Comment:
I'd call this test `constant_with_dependencies_on_constants.sol` and I think we should test more cases here:
- Dependency on another constant from the same contract.
- Dependency on inherited constants.
- Dependency on file-level constants.

Reply:
Done